mount/mount_windows.go

```diff
@@ -98,19 +98,32 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 		getSMBMountMutex.LockKey(source)
 		defer getSMBMountMutex.UnlockKey(source)
 
-		if output, err := newSMBMapping(allOptions[0], allOptions[1], source); err != nil {
+		var output string
+		var err error
+		username := allOptions[0]
+		password := allOptions[1]
+		if output, err = newSMBMapping(username, password, source); err != nil {
+			klog.Warningf("SMB Mapping(%s) returned with error(%v), output(%s)", source, err, string(output))
```

Comment:
will this output contain the username/password? if so, it must not be logged

Reply:
no, it won't, it uses `sensitiveOptionsRemoved` var in the error msg:
https://github.com/kubernetes/utils/blob/e0bba7c5b4d1bddbe8503553fe6d0389f06e2172/mount/mount_windows.go#L142-L144